### PR TITLE
proof-metrics

### PR DESCRIPTION
**Pull request by kaisbaccour**

### CODE REVIEW

File: lgn-provers/src/provers/v1/query/euclid_prover.rs [hunk 1]
```diff
@@ -92,12 +93,16 @@ impl StorageQueryProver for EuclidQueryProver {
         let input = QueryCircuitInput::Query(circuit_input);
         let proof = self.params.generate_proof(input)?;
 
+        let proof_type = "universal circuit";
```
Comment (hackaugusto) [inline lgn-provers/src/provers/v1/query/euclid_prover.rs:96]: We should replace the space with an underline:

```suggestion
        let proof_type = "universal_circuit";
```

I believe that spaces are not supported for labels by prometheus:

> Labels may contain ASCII letters, numbers, as well as underscores. They must match the regex [a-zA-Z_][a-zA-Z0-9_]*.

**Review by hackaugusto — APPROVED** [review]
would replace ` ` and `-` by `_` on all labels, otherwise LGTM

File: lgn-provers/src/provers/v1/query/euclid_prover.rs (hunk 1 above)
Comment (kaisbaccour) [reply lgn-provers/src/provers/v1/query/euclid_prover.rs:96]: Oh yeah they are not. well spotted

Comment (kaisbaccour) [reply lgn-provers/src/provers/v1/query/euclid_prover.rs]: Fixed

**Pull request merged**